Documentation/InstallOnMac.md

```diff
@@ -4,40 +4,36 @@
 ...or just tell more friends. As long as I'm maintaining this software, it will remain **free** and **open source**.
 
 
-
 # Run Libation on MacOS
 This walkthrough should get you up and running with Libation on your Mac.
 
 ## Install Libation
 
-- Download latest MacOS zip to downloads folder
-- Extract and rename folder to Libation
-- in terminal type cd and then drag your folder of libation to terminal so it looks like `cd/users/YourName/Downloads/Libation`
-- Type following commands
-
-```console
-chmod +x ./Libation
-sudo spctl --add --label "Libation" ./Libation
-./Libation
-```
-
-## Trouble with Gatekeeper?
-
-If Gatekeeper is giving you trouble with Libation:
+- Download the `Libation.app-x64-x.x.x.tgz` file from the latest release and extract it. 
```

Comment:
Thanks for re-adding 'macos'. Here also, plz

Reply:
Thanks. Done.